contracts/precompile-usages/PCUSortValidatorsByBeacon.sol

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.9;
+
+interface IPCUSortValidatorsByBeacon {
+  function requestSortValidatorSet(
+    uint256 beacon,
+    uint256 period,
+    uint256 numGovernanceValidator,
+    uint256 numStandardValidator,
+    uint256 numRotatingValidator,
+    address[] calldata ids,
+    uint256[] calldata stakedAmounts,
+    uint256[] calldata trustedWeights
+  ) external;
+
+  function pickValidatorSet(uint256 period, uint256 epoch) external view returns (address[] memory pickedValidatorIds);
+}
+
+abstract contract PCUSortValidatorsByBeacon {
+  /**
+   * @dev Returns the address of PCU contract.
+   *
+   * Note: This function is marked as virtual for being wrapping in mock contract for testing purpose.
+   */
+  function _getPCU() internal view virtual returns (IPCUSortValidatorsByBeacon) {
+    return IPCUSortValidatorsByBeacon(address(0x60));
```

Comment:
Confirm the address with Protocol team yet?

Reply:
removed since we won't use precompile contract anymore.